docs-client/src/containers/HomePage/index.tsx

```diff
@@ -13,12 +13,71 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Tooltip from '@material-ui/core/Tooltip';
 import Typography from '@material-ui/core/Typography';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
 import React from 'react';
+import Section from '../../components/Section';
+import { convertLongToUTCDate, Versions } from '../../lib/versions';
+
+interface OwnProps {
+  versions?: Versions;
+}
 
-export default class HomePage extends React.PureComponent {
+dayjs.extend(relativeTime);
+
+function formatReadableTime(dateTime: number) {
+  return dayjs().to(dayjs(dateTime));
+}
+type Props = OwnProps;
+
+export default class HomePage extends React.PureComponent<Props> {
   public render() {
+    const { versions } = this.props;
+    const data = versions;
+    const tableRows: any = [];
+
+    if (data) {
+      data.getVersions().forEach((version) => {
+        tableRows.push(
+          <TableRow>
+            <TableCell>{version.artifactId}</TableCell>
+            <TableCell>
+              {version.artifactVersion}
+              {data.getArmeriaArtifactVersion() !== version.artifactVersion
+                ? '(!)'
+                : ''}
+            </TableCell>
+            <TableCell>
+              <Tooltip title={convertLongToUTCDate(version.commitTimeMillis)}>
+                <Typography variant="body2">
+                  {formatReadableTime(version.commitTimeMillis)}
+                </Typography>
+              </Tooltip>
+            </TableCell>
+            <TableCell>
+              <Tooltip title={version.longCommitHash}>
+                <Typography variant="body2">
+                  <a
+                    href={`https://github.com/line/armeria/commit/${version.shortCommitHash}`}
```

Comment:
Nice idea! Shouldn't we use `longCommitHash` for disambiguation?

Reply:
Okay. I will change `shortCommitHash` to `longCommitHash`.
And I want to delete tooltip about `longCommitHash` because of don't use it.